error-codes.json

```diff
@@ -1,36 +1 @@
-{
-  "$schema": "https://schemas.exasol.com/error_code_report-1.0.0.json",
-  "projectName": "exasol-error-reporting",
-  "projectVersion": "0.4.0",
-  "errorCodes": [
-    {
-      "identifier": "E-ERP-1",
-      "message": "Invalid error code {{code}}.",
-      "messagePlaceholders": [
-        {
-          "placeholder": "code",
-          "description": "Error code which was causing the error."
-        }
-      ],
-      "mitigations": [
-        "Ensure you follow the standard error code format."
-      ],
-      "sourceFile": "_error.py"
-    },
-    {
-      "identifier": "E-ERP-2",
-      "message": "Unknown error/exception occurred.",
-      "messagePlaceholders": [
-        {
-          "placeholder": "traceback",
-          "description": "Exception traceback which lead to the generation of this error."
-        }
-      ],
-      "description": "An unexpected error occurred during the creation of the error",
-      "mitigations": [
-        "A good starting point would be to investigate the cause of the attached exception.\n\nTrackback:\n    {{traceback}}"
-      ],
-      "sourceFile": "_error.py"
-    }
-  ]
-}
\ No newline at end of file
+{"$schema": "https://schemas.exasol.com/error_code_report-1.0.0.json", "projectName": "exasol-error-reporting", "projectVersion": "0.5.0", "errorCodes": [{"identifier": "E-ERP-1", "message": "Invalid error code {{code}}.", "messagePlaceholders": [{"placeholder": "code", "description": "Error code which was causing the error."}], "mitigations": ["Ensure you follow the standard error code format."], "sourceFile": "_error.py"}, {"identifier": "E-ERP-2", "message": "Unknown error/exception occurred.", "messagePlaceholders": [{"placeholder": "traceback", "description": "Exception traceback which lead to the generation of this error."}], "description": "An unexpected error occurred during the creation of the error", "mitigations": ["A good starting point would be to investigate the cause of the attached exception.\n\nTrackback:\n    {{traceback}}"], "sourceFile": "_error.py"}]}
```

Comment:
are you sure this is a good idea, ah this is generated, in that case , mark it for github as generated